apps/prairielearn/src/lib/db-types.ts

```diff
@@ -84,8 +84,269 @@ export const DateFromISOString = z
   )
   .transform((s) => new Date(s));
 
-export const ModeSchema = z.enum(['Public', 'Exam', 'SEB']);
-export type Mode = z.infer<typeof ModeSchema>;
+/**
+ * Miscellaneous schemas; keep these alphabetized.
+ */
+
+export const AssessmentsFormatForQuestionSchema = z.array(
+  z.object({
+    label: z.string().nullable(),
+    assessment_id: IdSchema,
+    course_instance_id: IdSchema,
+    color: z.string().nullable(),
+  }),
+);
+
+// Result of grading_job_status sproc
+export const GradingJobStatusSchema = z.enum([
+  'none',
+  'canceled',
+  'queued',
+  'grading',
+  'graded',
+  'requested',
+]);
+export type GradingJobStatus = z.infer<typeof GradingJobStatusSchema>;
+
+/**
+ * Enum schemas; keep these alphabetized.
+ */
+
+export const EnumJobStatusSchema = z.enum(['Running', 'Success', 'Error']);
+export type EnumJobStatus = z.infer<typeof EnumJobStatusSchema>;
+
+export const EnumModeSchema = z.enum(['Public', 'Exam', 'SEB']);
+export type EnumMode = z.infer<typeof EnumModeSchema>;
```

Comment:
The only other change in this PR was changing `ModeSchema`/`Mode` to `EnumModeSchema`/`EnumMode` for consistency with the other enum types.